gui/autofish.lua

```diff
@@ -0,0 +1,110 @@
+-- config ui for autofish
+
+local gui = require("gui")
+local widgets = require("gui.widgets")
+local script = reqscript("autofish")
+
+local REFRESH_MS = 10000
+
+---
+-- Autofish
+---
+Autofish = defclass(Autofish, widgets.Window)
+Autofish.ATTRS = {
+    frame_title = "Autofish",
+    frame = {w=45, h=12},
+    resizable = false
```

Comment:
this should probably be true so players can squish your screen down to a minimum size so they can still see how much fish they have, but take up as little screen space as possible. Try setting `resize_min` to the minimum size that the screen is useful at.

Reply:
The screen is already as small as it can be and still be useful, without a minimal mode.